otherlibs/stdlib_alpha/effect.ml

```diff
@@ -1306,53 +1324,55 @@ module Make_rec (Ops : Operations_rec)
       
        We may be able to rewrite this to avoid magic. *)
       let f h a = f (Obj.magic_portable h) a [@nontail] in
-      let k : (a Modes.Portable.t, b, t, unit) continuation = Obj.magic (fiber f) in
+      let k : (a Modes.Portable.t, b, t, unit) continuation = Obj.magic (fiber f : _ @@ unique) in
```

Comment:
What is the `unique` annotation for here?

Reply:
Just for checking that the result is `unique`, but it's now in the interface.